src/ItemBadges/ItemBadges.tsx

```diff
@@ -44,10 +52,14 @@ const ItemBadges = ({
   isPublishedTooltip = 'Published',
   isPublic = false,
   isPublicTooltip = 'Public',
+  isCollapsible = false,
+  isCollapsibleTooltip = 'Collapsible',
+  showChatbox = false,
+  showChatboxTooltip = 'Chat',
```

Comment:
Should the tooltips get their values from translations?

Reply:
Yes the consumers of this component should set the tooltips with translated content. The default values are here in case this is used without translations. Maybe I should make the tooltip texts mandatory and not provide default values ?

Reply:
I think it's fine for now, but yes IMO the boundaries are not well-defined. The consumers still have to each redo the work for the translations in this instance, even though the semantics that are tied to the item models are enclosed from within the component.